src/ffi/methodobject.rs

```diff
@@ -30,6 +30,14 @@ pub type PyCFunctionWithKeywords = unsafe extern "C" fn(
     kwds: *mut PyObject,
 ) -> *mut PyObject;
 
+#[cfg(all(Py_3_7, not(Py_LIMITED_API)))]
```

Comment:
In `methodobject.h`, this is unconditionally defined (i.e. part of the limited API). (It should probably be considered CPython-specific, however)

```suggestion
#[cfg(all(Py_3_7))]
```